pmoves/services/hi-rag-gateway-v2/clients/qdrant.py

```
@@ -0,0 +1,223 @@
+            return
+
+    try:
+        if info is None or needs_recreate:
+            qdrant.recreate_collection(
+                collection_name=COLL,
+                vectors_config=VectorParams(size=vector_dim, distance=Distance.COSINE)
+            )
+            logger.info("(re)created Qdrant collection %s [dim=%d, metric=cosine]", COLL, vector_dim)
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)